assets/src/edit-story/components/canvas/karma/quickActions.karma.js

```diff
@@ -369,4 +366,160 @@ describe('Quick Actions integration', () => {
       ).toBe(false);
     });
   });
+
+  describe('background image selected', () => {
+    beforeEach(async () => {
+      await addBackgroundImage(0);
+
+      const {
+        state: {
+          currentPage: {
+            elements: [{ id }],
+          },
+        },
+      } = await fixture.renderHook(() => useStory());
+
+      const canvasElementWrapperId = fixture.querySelector(
+        `[data-testid="safezone"] [data-element-id="${id}"]`
+      );
+
+      await fixture.events.click(canvasElementWrapperId);
+    });
+
+    it(`clicking the \`${ACTION_TEXT.REPLACE_BACKGROUND_MEDIA}\` button should select select the media tab and focus the media tab`, async () => {
+      // change tab to make sure tab isn't selected before quick action
+      // hide 3p modal before we click the quick action
+      await fixture.events.click(fixture.editor.library.mediaTab);
+      // tab to dismiss button and press enter
+      await fixture.events.keyboard.press('tab');
+      await fixture.events.keyboard.press('tab');
+      await fixture.events.keyboard.press('Enter');
```

Comment:
🤔 I wonder if there's a util that would do this for us. I know I did this too but I'm sure others have run into this problem before too, right?

Reply:
It's only an issue when going to that tab, which most tests aren't. Could add a low priority task to make a util - it's a good thought, I'll make a ticket - gotta start thinking about our next fixit week ;)